modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -190,7 +192,11 @@ spec:
 
 [NOTE]
 ====
-There must be operating pods in the project before a VPA can determine recommended resources and apply the recommendations to new pods.
+For the VPA to provide you with an accurate recommendation, the VPA must observe pod metrics while the running workload uses resources that are not considered atypical. 
+
+For workloads with cyclical resource usage patterns, the VPA provides a more accurate recommendation after it observes an entire workload cycle. 
```

Comment:
It's not a workload cycle, it's a resource usage cycle. A workload cycle might be pods stopping and starting on some kind of cycle, and that's not what we want to convey.

As an example of what we are talking about, if your website is typically very busy on Monday mornings, but not especially busy at other times, the VPA needs to observe the resource usage for the entire week to be able to see its Monday usage. Otherwise, it might make a naive recommendation based upon the usage during the rest of the week when the website is mostly idle.

In my opinion, what you have is fine if you put "entire cycle" instead of "entire workload cycle" since the latter doesn't mean what we want to say. "workload's entire cycle" could also work, since it implies the type of cycle mentioned earlier (i.e. a cycle of resource usage).

Reply:
Thanks for the explanation. I went with "workload's entire cycle" as it strengthens the relationship with the word "cyclical".